content/documentation/getting-started/index.adoc

```diff
@@ -76,16 +76,17 @@ bash <(curl -L https://git.io/getLatestKialiOperator)
 
 This will pull the latest version of the Kiali Operator and Kiali itself, and install them using the default options. If Kiali was already installed, you will be prompted to remove the older version.
 
-icon:lightbulb[size=1x]{nbsp} You can configure the Kiali Operator install script by setting environment variables when you run it. For example, if you do not want the install script to install a Kiali custom resource (and thus avoid having the Kiali Operator install Kiali itself), set the OPERATOR_INSTALL_KIALI environment variable to "false" and then run the script. Read the comments at the top of the link:https://github.com/kiali/kiali/blob/master/operator/deploy/deploy-kiali-operator.sh[Kiali Operator install script] for details on what custom configuration settings are available to you.
+icon:lightbulb[size=1x]{nbsp} You can configure the Kiali Operator install script by setting environment variables when you run it. For example, if you do not want the install script to install a Kiali CR (and thus avoid having the Kiali Operator install Kiali itself), set the OPERATOR_INSTALL_KIALI environment variable to "false" and then run the script. Read the comments at the top of the link:https://github.com/kiali/kiali/blob/master/operator/deploy/deploy-kiali-operator.sh[Kiali Operator install script] for details on what custom configuration settings are available to you.
```

Comment:
we should probably add something like "or pass in the `--help` command line option for help documentation"